sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -438,6 +438,10 @@ object JdbcUtils extends Logging {
       (rs: ResultSet, row: InternalRow, pos: Int) =>
         row.setShort(pos, rs.getShort(pos + 1))
 
+    case ByteType =>
+      (rs: ResultSet, row: InternalRow, pos: Int) =>
+        row.update(pos, rs.getByte(pos + 1))
```

Comment:
Ur, it seems that we missed this. Can we use the following?
```scala
row.setByte(pos, rs.getByte(pos + 1))
```

Reply:
Ur, I missed that, too. I'll make a pr to fix this in master now.

Reply:
@dongjoon-hyun Can you check https://github.com/apache/spark/pull/26532?

Reply:
Sure, I approved it.